### PR TITLE
Deprecating Oracle Python integration (DBMON-3776)

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do? What is the motivation?

The Oracle Python integration will be deprecated in the next Agent release. This document preserves the configuration steps for the older Agent releases.

### Merge instructions

- [X] Please merge after reviewing

### CODE REVIEW

File: content/en/integrations/guide/_index.md
Comment (hestonhoffman) [inline content/en/integrations/guide/_index.md:1]: On line 72, let's change the desc to "Database guides"

File: content/en/integrations/guide/deprecated-oracle-integration.md [hunk 1]
```diff
@@ -0,0 +1,534 @@
+---
+title: Configuring Oracle integration on the Agent versions lower than 7.50.1
+kind: guide
+aliases:
+  - integrations/guide/deprecated-oracle-integration
```
Comment (hestonhoffman) [inline content/en/integrations/guide/deprecated-oracle-integration.md:5]:
```suggestion
```
Removing this one, cos it'll cause a circular redirect

File: content/en/integrations/guide/deprecated-oracle-integration.md [hunk 2]
```diff
@@ -0,0 +1,534 @@
...
+---
+
+# Oracle Integration
+
+![Oracle Dashboard][1]
```
Comment (hestonhoffman) [inline content/en/integrations/guide/deprecated-oracle-integration.md:10]:
```suggestion
# Oracle Integration

![Oracle Dashboard][1]
```
Let's remove the dashboard image. The guide probably just needs the old setup steps

File: content/en/integrations/guide/deprecated-oracle-integration.md [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+
+## Overview
+
+The Oracle integration provides health and performance metrics for your Oracle database in near real-time. Visualize these metrics with the provided [dashboard][21] and create monitors to alert your team on Oracle states.
+
+Enable [Database Monitoring][22] (DBM) for enhanced insights into query performance and database health. In addition to the standard integration, Datadog DBM provides query-level metrics, live and historical query snapshots, wait event analysis, database load, query explain plans, and blocking query insights.
```
Comment (hestonhoffman) [inline content/en/integrations/guide/deprecated-oracle-integration.md:16]:
```suggestion
This guide describes setting up the Oracle integration on versions of the Datadog Agent lower than 7.50.1. For more information on the Oracle integration, including setting it up on newer Agent versions, see the [Oracle integration documentation][1].
```

File: content/en/integrations/guide/_index.md [hunk 4]
```diff
@@ -69,11 +69,12 @@ cascade:
...
 {{< /whatsnext >}}
 
-{{< whatsnext desc="SQL guides:" >}}
+{{< whatsnext desc="Database guides:" >}}
     {{< nextlink href="integrations/guide/collect-more-metrics-from-the-sql-server-integration" tag=" SQL Server" >}}Collect more metrics from the SQL Server integration{{< /nextlink >}}
     {{< nextlink href="integrations/guide/collect-sql-server-custom-metrics" tag=" SQL Server" >}}Collect SQL Server custom metrics{{< /nextlink >}}
     {{< nextlink href="integrations/guide/use-wmi-to-collect-more-sql-server-performance-metrics" tag=" SQL Server" >}}Use WMI to collect more SQL Server performance metrics{{< /nextlink >}}
     {{< nextlink href="integrations/guide/connection-issues-with-the-sql-server-integration" tag=" SQL Server" >}}Connection issues with the SQL Server integration{{< /nextlink >}}
     {{< nextlink href="integrations/guide/mysql-custom-queries" tag=" MySQL" >}}MySQL custom queries{{< /nextlink >}}
+    {{< nextlink href="integrations/guide/deprecated-oracle-integration" tag=" Oracle" >}}Configuring Oracle integration on the Agent versions lower than 7.53{{< /nextlink >}}
```
Comment (jhgilbert) [inline content/en/integrations/guide/_index.md:78]:
```suggestion
    {{< nextlink href="integrations/guide/deprecated-oracle-integration" tag=" Oracle" >}}Configuring Oracle integration on the Agent versions lower than 7.50.1{{< /nextlink >}}
```

File: content/en/integrations/guide/deprecated-oracle-integration.md [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+Synonym created.
+
+
+SQL> select blocking_session,username,osuser, sid, serial#, wait_class, seconds_in_wait from v_$session
+where blocking_session is not NULL order by blocking_session;
+```
+
+3. Once configured, you can create a [monitor][10] based on `oracle.custom_query.locks` metrics.
+
+## Data Collected
```
Comment (jhgilbert) [inline content/en/integrations/guide/deprecated-oracle-integration.md:409]: The "Data Collected" section feels out of scope for the page, I recommend deleting it. They can read about the integration on its dedicated page.

**Review by jhgilbert — APPROVED** [review]
